languages/elixir/exercises/concept/enum/.docs/introduction.md

```diff
@@ -0,0 +1,60 @@
+`Enum` is a very useful module that provides a set of algorithms for working with enumerables. It offers sorting, filtering, grouping, counting, searching, finding min/max values, and much more.
+
+In Elixir, an enumerable is any data type that implements the `Enumerable` protocol. The most common of those are lists and maps.
+
+```elixir
+Enum.sort([4, 2, 3, 1])
+# => [1, 2, 3, 4]
+```
+
+```elixir
+Enum.empty?(%{})
+# => true
+```
```

Comment:
This does not introduce any new syntax, does it? If that is correct, this should be removed from the `introduction.md` file (it can be in the `after.md` document though).